### PR TITLE
Feature/issues

### PR DESCRIPTION
**Pull request by stewarty**
### All Submissions:

* [ ] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by paulschreiber — COMMENTED** [review]
I don't understand why the issues also have full/mini/wiki format options. That should be for profiles only.

**Review by paulschreiber — COMMENTED** [review]
Please use static classes instead of the singleton pattern. We have an open issue to get rid of this from existing classes; let's not add more.

File: template-parts/single-govpack_issues.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase
```
Comment (paulschreiber) [inline template-parts/single-govpack_issues.php:1]: This is almost identical to `single-govpack_profiles.php` and should be refactored to reduce duplication.

File: includes/CPT/class-issue.php [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+ * Register and handle the "Issue" Custom Post Type
+ */
+class Issue {
+
+	/**
+	 * Valid issue formats.
+	 *
+	 * @var array
+	 */
+	public static $issue_formats = [ 'full', 'mini', 'wiki' ];
```
Comment (paulschreiber) [inline includes/CPT/class-issue.php:22]: I don't think this applies to issues — it's for profiles. @designhawg do we have any designs for issue pages?

File: includes/Block/class-issue.php [hunk 3]
```diff
@@ -0,0 +1,90 @@
+<?php
```
Comment (paulschreiber) [inline includes/Block/class-issue.php:1]: This is very duplicate of class-profile.php. Construct a class-block.php base class and inherit from that.

File: includes/CPT/class-issue.php [hunk 4]
```diff
@@ -0,0 +1,252 @@
+<?php
```
Comment (paulschreiber) [inline includes/CPT/class-issue.php:1]: This is very duplicate of class-profile.php. Construct a class-post-type.php base class and inherit from that.

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: includes/CPT/class-issue.php (hunk 2 above)
Comment (leanderiversen) [reply includes/CPT/class-issue.php]: Formats have now been removed from "Issues".

File: template-parts/single-govpack_issues.php (hunk 1 above)
Comment (leanderiversen) [reply template-parts/single-govpack_issues.php]: This has been refactored into `single-govpack.php`, which checks for the current post type and pulls the correct stories.

File: includes/CPT/class-issue.php (hunk 4 above)
Comment (leanderiversen) [reply includes/CPT/class-issue.php:1]: This has been done now.

File: includes/Block/class-issue.php (hunk 3 above)
Comment (leanderiversen) [reply includes/Block/class-issue.php:1]: This has been done now.

**Pull request merged**